src/Exception/TubeNotFoundException.php

```diff
@@ -4,6 +4,12 @@
 
 namespace Pheanstalk\Exception;
 
+use Pheanstalk\Values\TubeName;
+
 final class TubeNotFoundException extends ClientException
 {
+    public function __construct(public readonly TubeName $tube)
```

Comment:
If you added a 'default' name - maybe something distinct like `[unknown]` (a local const?), then it wouldn't break backward compatibility with a new required parameter (and worse case, you are in no worse a condition that it is right now)

Looks like a useful addition otherwise